slothy/targets/arm_v81m/arch_v81m.py

```diff
@@ -36,6 +36,7 @@
 import inspect
 import re
 import math
+from abc import ABC, abstractmethod
```

Comment:
Can we hoist out the `Loop` class so we don't have to have copies in every architecture model? Maybe it could go to `helper.py` for now?